discovery/aws/lightsail.go

```diff
@@ -115,6 +115,7 @@ func (c *LightsailSDConfig) UnmarshalYAML(unmarshal func(interface{}) error) err
 
 		region, err := metadata.Region()
 		if err != nil {
+			// nolint:staticcheck
```

Comment:
Worth a comment why? (I think it might be a upper case first letter?) or just do `New("a Lightsail SD configuration...`

Reply:
Yes, uppercase first letter.

Reply:
I added a comment.